libclc/generic/lib/integer/ctz.cl

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <clc/clc.h>
+#include <clc/integer/clc_ctz.h>
+
+#define FUNCTION ctz
```

Comment:
Note the OpenCL builtin requires support for OpenCL 2.0 or newer. We should guard for that.

I think we can probably unconditionally include & compile the CLC `__clc_ctz` builtin so that we can use it internally if we wanted to, but we shouldn't include `ctz` in the OpenCL layer for OpenCL 1.X.

Reply:
thanks @frasercrmck 
`#if __OPENCL_C_VERSION__ >= CL_VERSION_2_0` guard is added to changes in libclc/generic in https://github.com/llvm/llvm-project/pull/135309/commits/361777ebdcd8f1d3ca59bfa82f0c79397f5a1845

__OPENCL_C_VERSION__ is 100 in libclc, so I created another PR to set OpenCL version to 3.0: https://github.com/llvm/llvm-project/pull/135733